src/reducers/__tests__/player.js

```diff
@@ -0,0 +1,114 @@
+import {ActionTypes, CellStates} from '../../constants';
+import reducer from '../player.js';
+
+const {INITIAL_STATE, BOARD_TILE_SELECTED, RACK_TILE_SELECTED, END_TURN} = ActionTypes;
+
+describe('src/reducers/player', function() {
+  describe('INITIAL_STATE', function() {
+    it('initializes a players rack', function() {
+      const state = reducer({}, {type: INITIAL_STATE, id: 1});
+
+      expect(state.id).toEqual(1);
+      expect(state.letters).toEqual(['a', 'b', 'c']);
+      expect(state.points).toEqual(0);
+      expect(state.selected).toEqual(-1);
+    });
+  });
+
+  describe('BOARD_TILE_SELECTED', function() {
+    it('removes the selected tile from a players rack', function() {
```

Comment:
this suite should have a test that places the tile on the board (unless you haven't gotten there with functionality yet).